app/controllers/fhir/r4/api_controller.rb

```diff
@@ -428,6 +438,7 @@ def all
     return if doorkeeper_authorize!(*QUESTIONNAIRE_RESPONSE_READ_SCOPES)
     return if doorkeeper_authorize!(*RELATED_PERSON_READ_SCOPES)
     return if doorkeeper_authorize!(*IMMUNIZATION_READ_SCOPES)
+    return if doorkeeper_authorize!(*PROVENANCE_READ_SCOPES)
```

Comment:
Github says no to me commenting on the line, but the comment at the top of `all` should be updated.